### ISSUE TITLE
Enhance Live Stream Experience 

### ISSUE THREAD

**Issue by lu4p** [libre-tube/LibreTube#917]
### Describe your suggested feature

The seekbar on currently running Livestreams should state only "live" when watching live.

If the user has an offset to live it should be indicated by a negative offset from live like this.
![Screenshot_20220730-022551](https://user-images.githubusercontent.com/46541492/181862939-a6c2d24c-70c7-488c-9dde-6a981b8c3b15.jpg)

Furthermore once "live" is reached the Playback speed should be reset to 1x.

On the feed/ channel page current live streams should have a length of "Live" on red background.

![Screenshot_20220730-023820](https://user-images.githubusercontent.com/46541492/181863632-0c35073f-5a48-4a17-bdfa-a10860a7b930.jpg)

Also no published date in the feed or "started x ago" instead of the Unix epoch.

Optimally there should also be the option to view the live chat, however Newpipe Extractor probably doesn't support this right now.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by lu4p** [comment]
Small suggestion: 
The player should play on e.g. 1.5x speed until all loaded chunks are played back, not until 10 seconds video is still left. Or if exhausting all chunks leads to loading/ stuttering have the seekbar "Live" limit at 10s and the speed slowdown at 5s.

Otherwise the player will switch between "Live" and "-0.11 • Live" all the time after watching at 1.5x speed.

**Comment by lu4p** [comment]
Also when offset from live pressing Live in  "-5:45 • Live" should jump to live.

**Comment by Bnyro** [comment]
Okay, these are good ideas, I'll improve that later :)

**Comment by Bnyro** [comment]
Feel free to join us on Matrix if you want to discuss about some things

**Comment by Bnyro** [comment]
And I forgot to mention that you were right, the NewPipeExtractor doesn't support the live chat of streams yet.

**Comment by Bnyro** [comment]
> Small suggestion: The player should play on e.g. 1.5x speed until all loaded chunks are played back, not until 10 seconds video is still left. Or if exhausting all chunks leads to loading/ stuttering have the seekbar "Live" limit at 10s and the speed slowdown at 5s.
> 
> Otherwise the player will switch between "Live" and "-0.11 • Live" all the time after watching at 1.5x speed.

We can't seek to the end of the stream directly because it would cause buffering but about 3s before or sth should work too. Just used 10s for testing.
